controllers/posts.go

```diff
@@ -138,15 +139,17 @@ func (pc *postController) UpdatePost(ctx echo.Context) error {
 	}
 
 	// Make a new updated post
-	categories := []models.Category{}
+	var categories []*models.Category
```

Comment:
Why?

Reply:
Both of these have been corrected, what is the problem?
In fact, this change that you mentioned already happened

Reply:
I had to make these minor changes to the syntax to fix the error